Automation_Test_Framework/Asserter/Asserter.cs

```diff
@@ -0,0 +1,168 @@
+﻿using Core_Framework.DriverCore;
+using FluentAssertions;
+using NUnit.Framework;
+using OpenQA.Selenium;
+using TheRookiesApp.DAO;
+
+namespace AssetManagementTestProject.Asserter;
+/// <summary>
+/// Assertions for strings/DAO/List DAO objects
+/// Also check order
+/// Alternatives if FluentAssertions fails
+/// </summary>
+public class Asserter : WebDriverBase
+{
+    public Asserter(IWebDriver driver) : base(driver)
+    {
+    }
+
+
+    // ------------------------------- CONVERTED JSON FLUENT ASSERTION -------------------------------
+
+    public static void AssertListStringEquals(string actual, string expected)
+    {
+        // Compare two deserialized JSON strings
+        AssertEquals(actual, expected);
+        //actual.Should().BeEquivalentTo(expected);
+    }
+    // ------------------------------- DAO LIST WITH FLUENT ASSERTION -------------------------------
+
+    public static void AssertUserListsEquals(List<UserListDAO> actual, List<UserListDAO> expected)
+    {
+        AssertEquals(actual, expected);
+        //actual.Should().BeEquivalentTo(expected);
```

Comment:
Is there any meaning to use the fluent assertion here?
You can use the hook instead to log HTML report. Don't do this way to just log the html log

Reply:
Not done yet. Remove comment unused commands.
Check other classes